src/Features/ExternalAccess/Copilot/Microsoft.CodeAnalysis.ExternalAccess.Copilot.csproj

```diff
@@ -29,6 +30,10 @@
     <ProjectReference Include="..\..\Core\Portable\Microsoft.CodeAnalysis.Features.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Threading" />
```

Comment:
this seems undesirable.  this will also make it so we can't use this code in vscode.  i dont' see anything that would make this necessary.